coordinator-common/src/main/java/org/apache/kafka/coordinator/common/runtime/CoordinatorRuntime.java

```diff
@@ -1775,7 +1775,7 @@ public void onHighWatermarkUpdated(
     /**
      * 16KB. Used for initial buffer size for write operations.
      */
-    static final int MIN_BUFFER_SIZE = 16384;
+    public static final int MIN_BUFFER_SIZE = 16384;
```

Comment:
Is this change necessary?

Reply:
No, it is only required within coordinator-common as of now